src/util/version.tsx

```diff
@@ -0,0 +1,2 @@
+export const RECENT_MAJOR_SERVER_VERSION = 5;
+export const UI_VERSION = "5-1";
```

Comment:
The hyphen here is a bit confusing, shouldn't it be a dot instead?
```suggestion
export const UI_VERSION = "5.1";
```

Reply:
I was thinking about it -- I assume it might be confusing to have two version string with "5." concatenated together. Hence my suggestion with the minus. See also the [proposed process](https://docs.google.com/document/d/15xqgRFLMHtLC-SehjW3m0OX-dXjAGn9ylvs0ZHFdrdM/edit) for the ui release.

Reply:
Ok, if everybody agrees with that I'm fine too, it's just that it's a bit different from what we discussed (and noted down) in the sync with the core team yesterday. Hence my questions (this and the one above).

Reply:
We decided to make this . based in the doc after all :)